src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/commands.py

```diff
@@ -6,47 +6,44 @@
 # pylint: disable=line-too-long
 # pylint: disable=too-many-statements
 
+from azure.cli.core.commands import CliCommandType
+
 
 def load_command_table(self, _):
-    from azure.cli.core.commands import CliCommandType
     from azure.cli.command_modules.eventhubs._client_factory import (namespaces_mgmt_client_factory,
                                                                      event_hub_mgmt_client_factory,
                                                                      consumer_groups_mgmt_client_factory,
                                                                      disaster_recovery_mgmt_client_factory)
-    from .custom import empty_on_404
+    from ._exception_handler import response_exception_handler, empty_on_404
 
     eh_namespace_util = CliCommandType(
         operations_tmpl='azure.mgmt.eventhub.operations.namespaces_operations#NamespacesOperations.{}',
-        client_factory=namespaces_mgmt_client_factory
-    )
+        client_factory=namespaces_mgmt_client_factory)
 
     eh_event_hub_util = CliCommandType(
         operations_tmpl='azure.mgmt.eventhub.operations.event_hubs_operations#EventHubsOperations.{}',
-        client_factory=event_hub_mgmt_client_factory
-    )
+        client_factory=event_hub_mgmt_client_factory)
 
     eh_consumer_groups_util = CliCommandType(
         operations_tmpl='azure.mgmt.eventhub.operations.consumer_groups_operations#ConsumerGroupsOperations.{}',
-        client_factory=consumer_groups_mgmt_client_factory
-    )
+        client_factory=consumer_groups_mgmt_client_factory)
 
     eh_geodr_util = CliCommandType(
         operations_tmpl='azure.mgmt.eventhub.operations.disaster_recovery_configs_operations#DisasterRecoveryConfigsOperations.{}',
-        client_factory=disaster_recovery_mgmt_client_factory
-    )
+        client_factory=disaster_recovery_mgmt_client_factory)
 
 # Namespace Region
     custom_tmpl = 'azure.cli.command_modules.eventhubs.custom#{}'
     eventhubs_custom = CliCommandType(operations_tmpl=custom_tmpl)
-    with self.command_group('eventhubs namespace', eh_namespace_util, client_factory=namespaces_mgmt_client_factory) as g:
+    with self.command_group('eventhubs namespace', eh_namespace_util, client_factory=namespaces_mgmt_client_factory, exception_handler=response_exception_handler) as g:
         g.custom_command('create', 'cli_namespace_create')
         g.show_command('show', 'get')
         g.custom_command('list', 'cli_namespace_list', exception_handler=empty_on_404)
```

Comment:
@v-Ajnava please remove the `empty_on_404` behavior for list commands. They should fail like our other list commands.

Reply:
resolved